packages/api-server/package.json

```diff
@@ -3,29 +3,31 @@
   "description": "Redwood's HTTP server for Serverless Functions",
   "version": "0.26.2",
   "bin": {
-    "api-server": "./dist/index.js"
+    "api-server": "./dist/index.js",
+    "rw-api-server": "./dist/index.js",
```

Comment:
Are api-server and rw-api-server different names for the same thing now?

Reply:
They're the same thing. I hope to deprecate `api-server` since it's a bit ambiguous, and I'm worried about bin namespace conflicts.

Reply:
Thought so, and then my next suggestion was going to be to look at argv[0] and print a depr. warning if it's /^api-server$/

Reply:
Great, suggestion, thanks!